deeppavlov/core/common/registry.json

```diff
@@ -11,8 +11,6 @@
   "char_splitter": "deeppavlov.models.preprocessors.char_splitter:CharSplitter",
   "conll2003_reader": "deeppavlov.dataset_readers.conll2003_reader:Conll2003DatasetReader",
   "cos_sim_classifier": "deeppavlov.models.classifiers.cos_sim_classifier:CosineSimilarityClassifier",
-  "data_fitting_iterator": "deeppavlov.core.data.data_fitting_iterator:DataFittingIterator",
-  "data_learning_iterator": "deeppavlov.core.data.data_learning_iterator:DataLearningIterator",
```

Comment:
Return default iterators please.
Their removal is caused by a bug that will be fixed in #595